firebase-storage/src/test/java/com/google/firebase/storage/UploadTest.java

```diff
@@ -128,56 +128,38 @@ public void fileUploadWith500() throws Exception {
     TestUtil.verifyTaskStateChanges("fileUploadWith500", task.getResult().toString());
   }
 
-  //  TODO(b/269131915): Fix this flaky test
-  //  @Test
-  //  public void cantUploadToRoot() throws Exception {
-  //    System.out.println("Starting test cantUploadToRoot.");
-  //
-  //    StorageReference storage =
-  //        FirebaseStorage.getInstance().getReferenceFromUrl("gs://fooey.appspot.com");
-  //
-  //    AtomicReference<Exception> taskException = new AtomicReference<>();
-  //
-  //    final UploadTask task = storage.putBytes(new byte[] {});
-  //
-  //    try {
-  //      task.getResult();
-  //      Assert.fail();
-  //    } catch (IllegalStateException ignore) {
-  //      // Task is not yet done.
-  //    }
-  //
-  //    Assert.assertNull(task.getException());
-  //
-  //    task.addOnFailureListener(
-  //        (exception) -> {
-  //          Assert.assertEquals(
```

Comment:
I think it's worth adding back the message check, as the exception thrown could be something else entirely, and this test would erroneously pass

Reply:
Ahh I was under the impression this was the only likely exception to be thrown. SGTM, will add an alternative check at the end- that's slightly less strict to be safe.